cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -647,6 +646,15 @@ def check_deployments(manager, old_deployments, logger,
     _log('Found correct deployments', logger, tenant)
 
 
+def check_status(manager):
+    status = manager.client.manager.get_status()
+    for service in status['services']:
+        for instance in service['instances']:
```

Comment:
We already have code to check this on the managers- <host>.verify_services_are_running(), which handles community as well by not failing when stage is not running on community.

Reply:
It's not exactly what I need. This function waits for vm creation

Reply:
It doesn't wait for VM creation, though it is also used for that.

If you want it without the retry it might be worth splitting the service status check there into a 'check_services' and then calling that, because otherwise you're still in the state where you're going to break community tests.